docs/reference/modules/threadpool.asciidoc

```diff
@@ -34,44 +34,44 @@ There are several thread pools, but the important ones include:
 `write`::
     For single-document index/delete/update and bulk requests. Thread pool type
     is `fixed` with a size of <<node.processors, `# of allocated processors`>>,
-    queue_size of `200`. The maximum size for this pool is `1 + `
+    queue_size of `200`. The maximum size for this pool is `pass:[1 + ]`
     <<node.processors, `# of allocated processors`>>.
```

Comment:
As-is, this leaves an annoying white space between `1 +` and `# of allocated processors`:

<img width="340" alt="Screen Shot 2020-04-10 at 1 34 49 PM" src="https://user-images.githubusercontent.com/40268737/79010562-3d945780-7b30-11ea-90e4-6b5880e95a4e.png">

I think this is because Asciidoctor adds a space for line breaks within a paragraph. You can get rid of it by
moving `pass:[1 + ]` to the following line:

```suggestion
    queue_size of `200`. The maximum size for this pool is 
    `pass:[1 + ]`<<node.processors, `# of allocated processors`>>.
```

<img width="341" alt="Screen Shot 2020-04-10 at 1 37 21 PM" src="https://user-images.githubusercontent.com/40268737/79010674-72081380-7b30-11ea-8f9e-ad051e42bfb3.png">

Not a big deal either way though.

Reply:
Oh, you're right, that extra whitespace is annoying!